hw3/people.data.js

```diff
@@ -1,48 +1,51 @@
-export const SarahMay = {
-  firstName: "Sarah",
-  lastName: "May",
-  birthDate: "13.03.1990",
-  address: {
-    country: "Spain",
-    city: "Seville",
-    street: "calle Verde",
-    house: "5",
-    apartment: "2A",
-  },
-  job: {
-    title: "teacher",
-    experience: 5
+
```

Comment:
remove this empty line, this is the reason why this file is marked as changed. Be careful next time